### PR TITLE
arp: add Spanish translation

### PR DESCRIPTION
**Pull request by teboarte**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/arp.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# arp
+
+> Muestra y Manipula la cache ARP del sistema.
+> Más information: <https://manned.org/arp>.
+
+- Muestra la tabla ARP actual:
+
+`arp -a`
+
+- Borra una entrada especifica:
```
Comment (kant) [inline pages.es/common/arp.md:10]: específica

**Review by kant — COMMENTED** [review]
minor fixes need it

**Review by kant — COMMENTED** [review]
minor fixes need it

File: pages.es/common/arp.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# arp
+
+> Muestra y Manipula la cache ARP del sistema.
```
Comment (tricantivu) [inline pages.es/common/arp.md:3]:
```suggestion
> Muestra y manipula la caché ARP del sistema.
```

[^1]: https://dle.rae.es/memoria#SdpzSSz

File: pages.es/common/arp.md (hunk 1 above)
Comment (tricantivu) [inline pages.es/common/arp.md:10]: As @kant suggested:

```suggestion
- Borra una entrada específica:
```

File: pages.es/common/arp.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tricantivu) [reply pages.es/common/arp.md:3]: Why remove `del sistema`?

**Review by spageektti — APPROVED** [review]

**Pull request merged**